tests/file_storage/tests.py

```diff
@@ -123,6 +128,112 @@ def test_file_access_options(self):
         self.storage.delete('storage_test')
         self.assertFalse(self.storage.exists('storage_test'))
 
+    def _test_file_time_getter(self, getter):
+        # We want to check for correct behaviour under both
+        # USE_TZ=True and USE_TZ=False. The tests are
+        # annoyingly similar, since they both set up a
+        # situation where system, Django and UTC are distinct.
+        self._test_file_time_getter_tz_handling_on(getter)
+        self._test_file_time_getter_tz_handling_off(getter)
+
+    @override_settings(USE_TZ=True, TIME_ZONE='Africa/Algiers')
```

Comment:
This test won't pass when pytz isn't installed. For consistency with the timezone tests, you should declare:

```python
requires_pytz = skipIf(pytz is None, "this test requires pytz")
```
and then decorate it with `@requires_pytz`.

This is a minor concern since the docs now encourage installing all optional dependencies before running the test suite, but I suppose it could save some headaches to people running the test suite without a virtualenv.
 on or systems that don't have `time.tzset` (that is, Windows). Have